provision/core/mariadb/provision.sh

```diff
@@ -114,14 +114,22 @@ function mysql_setup() {
   chmod 0644 "/home/vagrant/.my.cnf"
   vvv_info " * Copied /srv/provision/core/mariadb/config/root-my.cnf          to /home/vagrant/.my.cnf"
 
+  # this file should been obsolete, but somehow mariadb init.d script use it
+  cp -f  "/srv/provision/core/mariadb/config/debian.cnf" "/etc/mysql/debian.cnf"
+  vvv_info " * Copied /srv/provision/core/mariadb/config/debian.cnf          to /etc/mysql/debian.cnf"
+
+  # Due to systemd dependencies, in docker, mysql service is not auto started
+  vvv_info " * Force Restarting mysql service"
+  service mariadb restart
+
   if [ "${VVV_DOCKER}" != 1 ]; then
     check_mysql_root_password
   fi
 
   # MySQL gives us an error if we restart a non running service, which
   # happens after a `vagrant halt`. Check to see if it's running before
   # deciding whether to start or restart.
-  if [ $(service mariadb status|grep 'mysql start/running' | wc -l) -ne 1 ]; then
+  if [ $(service mariadb status|grep 'Uptime' | wc -l) -ne 1 ]; then
```

Comment:
this doesn't work on Parallels based VMs and maybe others, would it not be better to check for `running` or `active`?